judo-tatami-jsl-jsl2psm/src/test/java/hu/blackbelt/judo/tatami/jsl/jsl2psm/actor/JslModel2PsmAnonymousActorTest.java

```diff
@@ -0,0 +1,220 @@
+        		.transferName("UserTransfer")
+        		.operationName("default")
+        		.behaviour(TransferOperationBehaviourType.GET_TEMPLATE)
+        		.isBound(false)
+        		.outputType("UserTransfer")
+        		.isMappedOutputType(true));
+
+        assertCrudOperation(params()
+        		.transferName("Actor")
+        		.operationName("listOfManager")
+        		.behaviour(TransferOperationBehaviourType.LIST)
+        		.isBound(false)
+        		.inputType("_UserTransferQueryCustomizer")
+        		.isMappedInputType(false)
+        		.outputType("UserTransfer")
+        		.isMappedOutputType(true));    
+
+        assertCrudOperation(params()
+        		.transferName("Actor")
+        		.operationName("createInstanceOfManager")
+        		.behaviour(TransferOperationBehaviourType.CREATE_INSTANCE)
+        		.isBound(false)
+        		.inputType("UserTransfer")
+        		.isMappedInputType(true)
+        		.outputType("UserTransfer")
+        		.isMappedOutputType(true));  
+
+        assertCrudOperation(params()
+        		.transferName("Actor")
+        		.operationName("validateCreateInstanceOfManager")
+        		.behaviour(TransferOperationBehaviourType.VALIDATE_CREATE)
+        		.isBound(false)
+        		.inputType("UserTransfer")
+        		.isMappedInputType(true)
+        		.outputType("UserTransfer")
+        		.isMappedOutputType(true));   
+    }
+    
+    
+    @Builder
+    @Getter
+    private static class AssertCrudOperationParameters {
+
+    	String transferName;
+    	String operationName;
+    	TransferOperationBehaviourType behaviour;
+    	boolean isBound;
+		String inputType;
+		boolean isMappedInputType;
+		String outputType;
+		boolean isMappedOutputType;
+    }
+    
+    private static AssertCrudOperationParameters.AssertCrudOperationParametersBuilder params() {
+    	return AssertCrudOperationParameters.builder();
+    }
+    
+    private void assertCrudOperation(AssertCrudOperationParameters.AssertCrudOperationParametersBuilder p) {
+
+        TransferOperation operation = assertTransferObjectOperation(p.transferName, p.operationName);
+        if (p.isBound) {
+        	assertThat(operation, instanceOf(BoundTransferOperation.class));        	
+        } else {
+        	assertThat(operation, instanceOf(UnboundOperation.class));        	        	
+        }
+    	assertThat(operation.getBehaviour().getBehaviourType(), equalTo(p.behaviour));
+        
+        if (p.inputType != null) {
+            assertThat(operation.getInput(), is(notNullValue()));
+            if (p.isMappedInputType) {
+            	assertThat(operation.getInput().getType(), equalTo(assertMappedTransferObject(p.inputType)));            	
+            } else {            	
+            	assertThat(operation.getInput().getType(), equalTo(assertUnmappedTransferObject(p.inputType)));            	
+            }
+            // assertThat(operation.getInput().getCardinality().getLower(), equalTo(1));
```

Comment:
👀

Reply:
Add some ticket for the commented test cases